js/test/date/testMeridiems.js

```diff
@@ -2117,5 +2125,34 @@ module.exports.testmeridiems = {
         test.equal(fmt[0].name, 'AM');
         test.equal(fmt[1].name, 'PM');
         test.done();
+    },
+    testMeridiem_pa_IN: function(test) {
+        test.expect(3);
+        var fmt = DateFmt.getMeridiemsRange({locale:"pa-IN"});
+        test.ok(fmt !== null);
+
+        test.equal(fmt[0].name, 'ਪੂ.ਦੁ.');
+        test.equal(fmt[1].name, 'ਬਾ.ਦੁ.');
+        test.done();
+    },
+    testMeridiem_pa_IN_Custom: function(test) {
+        if (ilib._getPlatform() !== "nodejs" || !ilib._dyndata || !ilib._dyncode) {
+            test.done();
+            return;
+        }
+        var multiPath = path.relative(process.cwd(), path.resolve(__dirname, "./custom"));
+        var ilibLoader = ilib.getLoader();
+        
+        ilibLoader.addPath(multiPath);
+
```

Comment:
Need a test for multiple multiPaths

Reply:
Yeah, Actually I assume the added path would be one. I'll add more test cases while updating codes